pkg/operator/ceph/pool/validate.go

```diff
@@ -67,6 +67,9 @@ func ValidatePoolSpec(context *clusterd.Context, clusterInfo *client.ClusterInfo
 		if !found {
 			return errors.Errorf("unrecognized failure domain %s", p.FailureDomain)
 		}
+	} else {
+		// If empty let's populate it with the default
+		p.FailureDomain = cephv1.DefaultFailureDomain
```

Comment:
In general, the pattern has been to leave the settings in the CR types so we can tell the difference between not set and set. For example, in some scenarios such as stretch cluster I could imagine the default should be zone failure domain. We may not actually want to change the default, not sure yet, but if we always set the default, we would be prevented from doing that in the future. Let's discuss.

Reply:
Let's discuss it!

Reply:
As per discussion, the defaulting was already present so this is ok.